mxnet-bot/EmailBot/EmailBot.py

```diff
@@ -0,0 +1,304 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import print_function
+from botocore.exceptions import ClientError
+from botocore.vendored import requests
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+import boto3
+import boto3.s3
+import datetime
+import logging
+import os
+import re
+
+logging.basicConfig(level=logging.INFO)
+logging.getLogger('boto3').setLevel(logging.CRITICAL)
+logging.getLogger('botocore').setLevel(logging.CRITICAL)
+
+LOGGER = logging.getLogger(__name__)
+
+
+class EmailBot:
+
+    # GitHub account credentials
+    GITHUB_USER = os.environ.get("GITHUB_USER")
+    GITHUB_OAUTH_TOKEN = os.environ.get("GITHUB_OAUTH_TOKEN")
+    AUTH = (GITHUB_USER, GITHUB_OAUTH_TOKEN)
+    REPO = os.environ.get("REPO")
+    # Send emails using Amazon Simple Email Service(SES)
+    # Sender's email address must be verified.
+    # Replace it with sender's email address
+    sender = "sender@email.com"
```

Comment:
Can we also set this as an `Environment Variable` instead of dig into the code and fix it?
Does the user required to use Gmail to send Email, can MS Email also using the same tool?

Reply:
Yes, every email address verified in SES can use this tool